### PR TITLE
Yasir/job card

### PR DESCRIPTION
**Pull request by YasirGaji**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 # testing
 /coverage
-
+.history
```
Comment (adenekan41) [inline .gitignore:11]: What's this for ?

File: .hintrc [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (adenekan41) [inline .hintrc:1]: Hmmm, whats this ??

File: public/images/jobs/index.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import CompanyLogo from './Company Logo.png'
```
Comment (adenekan41) [inline public/images/jobs/index.ts:1]: Nope, we already have svgr for this so you can simply do 



Also never not accentuate your file name without special chars 

```js
//e.g 

// Compay Logo.png

becomes 
// company_logo.png
or 
// company-logo.png

```

File: src/components/seo/index.ts [hunk 4]
```diff
@@ -1 +1,2 @@
 export * from './seo';
+export * from './jobcard/JobCard'
```
Comment (adenekan41) [inline src/components/seo/index.ts:2]: Christ ??

File: src/components/seo/jobcard/JobCard.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
Comment (adenekan41) [inline src/components/seo/jobcard/JobCard.tsx:1]: What is job card looking for in seo/jobcard ?? 

this is a private / constrainted component so it should easily live inside of 

dashboard/views/homes > a new folder called components

File: src/components/seo/jobcard/JobCard.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import Image from 'next/image';
+import { CallToAction, Flex, Heading, Text } from '@labs/components';
+import { CompanyLogo } from '../../../../public/images/jobs';
```
Comment (adenekan41) [inline src/components/seo/jobcard/JobCard.tsx:4]: Please delete

File: src/components/seo/jobcard/JobCard.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+          <Image src={CompanyLogo} alt='Company Logo' />
+
+          <div className={styles.info}>
+            <Heading.h6>Brand Designer</Heading.h6>
+            <Text color='var(--text-gray)'>Dribble. California. $120k - $140k</Text>
+          </div>
+        </div>
+				
+        <div className={styles.actionButtons}>
+          <button className={styles.save}>Save</button>
```
Comment (adenekan41) [inline src/components/seo/jobcard/JobCard.tsx:23]: Please use the private `CallToAction` component

File: src/components/seo/jobcard/JobCard.tsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Image from 'next/image';
+import { CallToAction, Flex, Heading, Text } from '@labs/components';
+import { CompanyLogo } from '../../../../public/images/jobs';
+import styles from './jobcard.module.scss'
+
+const JobCard = () => {
+  return (
+    <div className={styles.jobCard}>
```
Comment (adenekan41) [inline src/components/seo/jobcard/JobCard.tsx:9]: No camel case. 

styles.jobCard should be styles.JobCard 

looks a lot elegant that way

File: src/components/seo/jobcard/jobcard.module.scss [hunk 9]
```diff
@@ -0,0 +1,88 @@
+@use '@/styles/base/utils' as *;
+
+
+.jobCard {
+  @media (min-width: 768px) {
+    width: 1044px;
+  }
```
Comment (adenekan41) [inline src/components/seo/jobcard/jobcard.module.scss:7]: Nope ?? never do this. ever 

```suggestion
width: 100%; 
```

does the trick

File: src/components/seo/jobcard/jobcard.module.scss [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 1.25rem; 
+  padding: 1.25rem; 
+  justify-content: space-around;
+  // border: 1px solid;
+  border-radius: 0.375rem;
+  padding-bottom: 1.7rem;
```
Comment (adenekan41) [inline src/components/seo/jobcard/jobcard.module.scss:15]: please lets refrain `rem` and i hate it let's use px and use the rem wrapper i have in place so 

```scss
gap: rem(14px)
```

File: src/components/seo/jobcard/jobcard.module.scss [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+
+  .jobDetails {
+    display: flex;
+    justify-content: space-between;
+    @media (min-width: 768px) {
+      width: 100%; 
+    }
```
Comment (adenekan41) [inline src/components/seo/jobcard/jobcard.module.scss:22]: WHy ???

File: src/components/seo/jobcard/jobcard.module.scss [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+    gap: 1.25rem;
+    
+
+    .logoAndInfo {
+      display: flex;
+      @media (min-width: 768px) {
+        height: 56px; 
+        width: 322px;
+      }
```
Comment (adenekan41) [inline src/components/seo/jobcard/jobcard.module.scss:31]: Whats with the sparring `media` query ??

File: src/components/seo/jobcard/jobcard.module.scss [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+      .info {
+        display: flex;
+        flex-direction: column;
+        width: 248px;
+        height: 54px;
+
+        h6 {
+          font-weight: 600;
+          font-size: 1.125rem;
+        }
```
Comment (adenekan41) [inline src/components/seo/jobcard/jobcard.module.scss:43]: Nope never hard code `font-size` we have a declarative direction

File: src/components/seo/jobcard/jobcard.module.scss [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+        border-radius: 9999px;
+        width: 79px;
+        height: 42px;
+      }
+
+      .apply {
+        border: 1px solid #3198F8;
+        background-color: #3198F8;
+        color: white;
+        border-radius: 9999px; /* Rounded to full for circular shape */
```
Comment (adenekan41) [inline src/components/seo/jobcard/jobcard.module.scss:63]: 50px is just about fine here, but they won't be any need for this cus we are using the actual buttons i created

File: src/features/dashboard/views/home/home.tsx [hunk 15]
```diff
@@ -22,6 +24,12 @@ export const DashboardHome = () => {
 				</Text>
 			</Flex.Column>
 			<SetupChecklist />
+
+			<Heading.h5 color="var(--text-black)" className='pt-10 pl-6'>
+				Random Job Based On Profile
+			</Heading.h5>
+			<JobCard />
```
Comment (adenekan41) [inline src/features/dashboard/views/home/home.tsx:31]: Lets just have a structural data for the job cards and map it e.g 

```tsx
const JOBS = [
	{
		...blah blah
	}
]

and you map below 
```

**Review by adenekan41 — CHANGES_REQUESTED** [review]
Let's fix these

File: .gitignore (hunk 1 above)
Comment (YasirGaji) [reply .gitignore:11]: Fixeed

File: .hintrc (hunk 2 above)
Comment (YasirGaji) [reply .hintrc:1]: fixed

File: public/images/jobs/index.ts (hunk 3 above)
Comment (YasirGaji) [reply public/images/jobs/index.ts:1]: Corrected

File: src/components/seo/index.ts (hunk 4 above)
Comment (YasirGaji) [reply src/components/seo/index.ts:2]: Fixed

File: src/components/seo/jobcard/JobCard.tsx (hunk 5 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/JobCard.tsx:1]: corrected

File: src/components/seo/jobcard/JobCard.tsx (hunk 6 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/JobCard.tsx:4]: Done

File: src/components/seo/jobcard/JobCard.tsx (hunk 7 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/JobCard.tsx:23]: done

File: src/components/seo/jobcard/JobCard.tsx (hunk 8 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/JobCard.tsx:9]: Done

File: src/components/seo/jobcard/jobcard.module.scss (hunk 9 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/jobcard.module.scss:7]: Done

File: src/components/seo/jobcard/jobcard.module.scss (hunk 10 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/jobcard.module.scss:15]: fixed

File: src/components/seo/jobcard/jobcard.module.scss (hunk 12 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/jobcard.module.scss:31]: corrected

File: src/components/seo/jobcard/jobcard.module.scss (hunk 13 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/jobcard.module.scss:43]: Corrected

File: src/components/seo/jobcard/jobcard.module.scss (hunk 14 above)
Comment (YasirGaji) [reply src/components/seo/jobcard/jobcard.module.scss:63]: Corrected

File: src/features/dashboard/views/home/home.tsx (hunk 15 above)
Comment (YasirGaji) [reply src/features/dashboard/views/home/home.tsx:31]: Done

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+        flex-direction: column;
+        width: 248px;
+        height: 54px;
+      }
+    }
+
+    .ActionButtons {
+      display: flex;
+      justify-content: space-between;
+      width: 177px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:38]: Why ?

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+
+      .Save {
+        border: 1px solid #b7b7b7 !important;
+        border-radius: 50px;
+        width: 79px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:43]: Why ?

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+        height: 42px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:44]: Why ?

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+      }
+
+      .Apply {
+        border: 1px solid #3198F8;
+        background-color: #3198F8;
+        color: white;
+        border-radius: 50px; 
+        width: 90px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:52]: Why ?

these things are already set by the directive component

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+        height: 42px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:53]: Why ?

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+      }
+    }
+
+  }
+
+  .Tags {
+    width: 283px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:60]: Why ?

always set dynamic w x h with properties like "padding"

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+    height: 32px;
+    display: flex;
+    justify-content: space-between;
+
+    .Tag {
+      width: 87px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:66]: Why ?

Stop setting hard width and height

File: .hintrc (hunk 2 above)
Comment (adenekan41) [reply .hintrc:1]: Please remove this file. and remove it from your cache

File: .gitignore (hunk 1 above)
Comment (adenekan41) [reply .gitignore:11]: i still see `.history`

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss [hunk 23]
```diff
@@ -0,0 +1,76 @@
...
+      height: 32px;
```
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:67]: Why ?

File: src/features/dashboard/views/home/component/jobcard/jobcard.module.scss (hunk 19 above)
Comment (adenekan41) [inline src/features/dashboard/views/home/component/jobcard/jobcard.module.scss:52]: you don't need to set these

**Pull request closed without merging**